src/components/search/Search.jsx

```diff
@@ -144,12 +144,12 @@ const Search = (props) => {
                        placeholder="Enter query string"
                        value={ queryString } />
                 <span className="input-group-btn">
-                  <button className="btn btn-default"
+                  <button className="btn btn-primary"
                           type="submit"
                           title="Submit search"
                           aria-label="Submit search"
                           data-testid="Submit search">
-                    <FontAwesomeIcon className="fa-search" icon={faSearch} />
+                    Search
```

Comment:
I'm unclear from the ticket. Are we supposed to replace faSearch with a blue button that says "search" or make faSearch a blue button? @astridu